lib/configure.js

```diff
@@ -334,6 +341,22 @@ module.exports = function() {
       if (err.message === 'repository not found') {
         process.stdout.write('I didn\'t find your repository in Travis.  Syncing Travis with GitHub…\n');
         return travis.users.sync.post()
+        .catch(function(err) {
+          // Ignore the exception if Travis is already syncing. The status code
+          // of the error is 409 and its message is 'Sync already in progress.
+          // Try again later.', but instead of relying on that we simply perform
+          // an explicit check.
+          return travisIsSyncing().then(function(is_syncing) {
```

Comment:
Nit: this function takes an *is_syncing* parameter, but the equivalent one above takes *isSyncing*. They should use the same format.